### DISCUSSION TITLE
Introduction and Mentorship

### DISCUSSION THREAD

**Discussion by gyandroid** [OpenMined/courses#135] [Introduce Yourself]
Hi Team,

I'm Gyan from the US and a privacy, open data and open governance enthusiast.
I got introduced to OpenMined and the courses they offer recently and this is the most exhaustive, structured material and community about Privacy I've seen. Thanks Andrew and team. Looking forward to learning more and contributing.

Also I'd love to sign up in the technical mentorship program once it opens up again and looking forward to enjoying the courses as well.

Thanks and happy 2021!

Gyan Radhakrishnan.

**Comment by yemikifouly** [comment]
Hello Gyan! 👋 Welcome to OpenMined! I am really glad you are liking the first course so far. Remember to post any question you might have in the board so that we can help. 🙂 

About the mentorship at OpenMined... We don't actually offer 1-on-1 technical mentorship. Our 1-on-1 mentorship program is non-technical and aimed at helping you navigate the community and reach the goal you set for yourself upon joining. The technical mentors that you will have access to during the course will interact with you and other students here on the board and through your project submissions that they will review. 🙂 

I hope all this makes sense to you. 🙂 If you are interested in our non-technical 1-on-1 mentorship program I will be sure to inform you once they start accepting mentees again.

**Reply by Onyinye91-ctrl** [reply]
I am also interested in the Mentorship Program. Please do inform me when you start accepting mentees @yemikifouly

**Reply by yemikifouly** [reply]
@Onyinye91-ctrl Will do Sandra! 🙂

**Reply by Onyinye91-ctrl** [reply]
Thank You

**Reply by kpuwal** [reply]
Hello @yemikifouly 🙂 would it be possible to inform me about the non-technical Mentorship Program as well?

**Reply by yemikifouly** [reply]
@kpuwal I will Kasia! 🙂

**Reply by kpuwal** [reply]
Thank you! 🙂
